.travis.yml

```diff
@@ -7,3 +7,4 @@ rvm:
   - 2.1.0
   - 2.2.0
   - 2.3.0
+before_script: bundle exec rake build && [[ -z $(git status -s) ]]
```

Comment:
Yeah it would work, not sure why I didn't think of it because it's very obvious. `git status -s` returns 0 either way. `before_script` wants a 0 if it should continue, or non-zero if it should fail. So that's the point of using `-z` - if `git status -s` outputs anything, return 1, and make `before_script` fail. 👍